website/api/controllers/create-or-update-one-newsletter-subscription.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+
+
+  friendlyName: 'Create or update one newsletter subscription',
+
+
+  description: 'Creates or updates a NewsletterSubscription record for the provided email address',
+
+
+  inputs: {
+    emailAddress: {
+      type: 'string',
+      required: true,
+    },
+
+    subscribeTo: {
+      type: 'string',
+      required: true,
+      description: 'The type of content that this user is changing their subscription for',
+      isIn: ['releases']
+    }
+  },
+
+
+  exits: {
+
+    success: {
+      description: 'A user has successfully changed their subscription to the Fleet newsletter'
+    },
+
+    emailAlreadyInUse: {
+      description: 'A user who is already subscribed to the newsletter'
```

Comment:
Include `statusCode` or `responseType` (in this case, statusCode, since there's no custom response for this, which is fine)

Example: https://github.com/fleetdm/fleet/blob/e335e835c93b3d29a10e99e9bb5059ed2da8000e/website/api/controllers/account/update-profile.js#L34

Why?  Because this way we won't send a 5xx response (triggering alerts) for something that is a recognized error.

Status codes:
- 4xx == expected error (something wrong that Fleet has anticipated and that the user might be able to fix and that the UI might want to show a special custom error for.)
- 5xx == unexpected error (something wrong, Fleet's fault, notify Fleet ASAP so that we can jump in and provide excellent, tailored support, fix it, and improve so it doesn't happen again)

Reply:
Except... wait... this isn't actually used below.  So it shouldn't be an exit at all.

Reply:
I removed the `emailAlreadyInUse` exit.